ui/tests/e2e/test.js

```diff
@@ -16,6 +17,9 @@ describe("End-to-end", () => {
     // Click first Age facet value
     let facetValueRow = await getFacetValueRow("Age", "10-19");
     await facetValueRow.click("input");
+    // After data is returned from backend, Smoker facet will have "107".
+    // See #63 for why we can't simply wait for div.grayText.
+    await page.waitForXPath("//text()[contains(.,'107')]");
```

Comment:
Instead of looking for a random value that's present in the new xpath but not the old one, we could `await totalHeaderCount = 137` and have that serve as the assertion.

Reply:
Done